src/app/_tests/mocks.ts

```diff
@@ -405,20 +407,17 @@ export const mockTractPage = (
     id: '1',
     position,
     parentPage: null,
-    parentPageParams: null,
-    nextPage: null,
-    previousPage: null,
     isHidden: false,
     _modals: null,
     findModal: null,
     _cards: null,
     visibleCards: null,
     backgroundImage: null,
     getAnalyticsEvents: null,
-    _dismissListeners: null,
-    _listeners: null,
-    dismissListeners: null,
-    listeners: null,
+    _dismissListeners: dismissListeners,
+    _listeners: listeners,
```

Comment:
I think you want to leave these as null for the mocks. they are internal data structures that I couldn't hide from the typescript headers, it was causing a compile/test error when I fully hid them previously. The actual listeners should be always be exposed via `dismissListeners` and `listeners`

I believe setting these to null will hopefully fix the current test errors, I'm also re-examining if I can truly hide them now that we are on a newer version of Kotlin